src/modules/user/services/user.service.ts

```diff
@@ -1,6 +1,37 @@
-import { Injectable } from '@nestjs/common';
+import { BadRequestException, Inject, Injectable } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import type { UserDTO } from '../dto/user.dto';
+import type { UserEntity } from '../entities/user.entity';
```

Comment:
типи повинні бути знизу
```suggestion
import { BadRequestException, Inject, Injectable } from '@nestjs/common';
import { Repository } from 'typeorm';
import { UserRepository } from '../repositories/user.repository';
import { UserService } from './user.service.abstract';
import { genSalt, hash } from 'bcrypt';

import type { UserDTO } from '../dto/user.dto';
import type { UserEntity } from '../entities/user.entity';

```